File: Assets/VirtualCity/ProtoDefine/Java/system/PlayerGameData.java
package com.kingston.jforgame.server.game.system.entity;

import com.kingston.jforgame.server.game.building.entity.Devlopments;
import com.kingston.jforgame.server.game.building.entity.House;
import com.kingston.jforgame.server.game.building.entity.HouseParts;
import com.kingston.jforgame.server.game.building.entity.Land;

import java.util.List;
import java.util.Map;

public class PlayerGameData {

    private volatile Map<String,Devlopments> devlopmentMap;

    private volatile Map<String, Land> landMap;

    private volatile Map<String, House> houseMap;

    private volatile List<HouseParts> housePartList;

    public Map<String, Devlopments> getDevlopmentMap() {
        return devlopmentMap;
    }

    public void setDevlopmentMap(Map<String, Devlopments> devlopmentMap) {
        this.devlopmentMap = devlopmentMap;
    }

    public Map<String, Land> getLandMap() {
        return landMap;
    }

    public void setLandMap(Map<String, Land> landMap) {
        this.landMap = landMap;
    }

    public Map<String, House> getHouseMap() {
        return houseMap;
    }

    public void setHouseMap(Map<String, House> houseMap) {
        this.houseMap = houseMap;
    }

    public List<HouseParts> getHousePartList() {
        return housePartList;
    }

    public void setHousePartList(List<HouseParts> housePartList) {
        this.housePartList = housePartList;
    }
}

File: Assets/VirtualCity/ProtoDefine/PlayerGameData.cs
using System.Collections.Generic;
using ProtoBuf;
namespace ProtoDefine {
[ProtoContract]



public class PlayerGameData {

    public  Dictionary<string,Devlopments> devlopmentMap;

    public  Dictionary<string, Land> landMap;

    public  Dictionary<string, House> houseMap;

    public  List<HouseParts> housePartList;

    public Dictionary<string, Devlopments> getDevlopmentMap() {
        return devlopmentMap;
    }

    public void setDevlopmentMap(Dictionary<string, Devlopments> devlopmentMap) {
        this.devlopmentMap = devlopmentMap;
    }

    public Dictionary<string, Land> getLandMap() {
        return landMap;
    }

    public void setLandMap(Dictionary<string, Land> landMap) {
        this.landMap = landMap;
    }

    public Dictionary<string, House> getHouseMap() {
        return houseMap;
    }

    public void setHouseMap(Dictionary<string, House> houseMap) {
        this.houseMap = houseMap;
    }

    public List<HouseParts> getHousePartList() {
        return housePartList;
    }

    public void setHousePartList(List<HouseParts> housePartList) {
        this.housePartList = housePartList;
    }
}
}